spec/lib/email/sender_spec.rb

```diff
@@ -738,13 +748,13 @@ def stub_deliver_response(message)
             "multipart/mixed; boundary=\"#{summary.body.boundary}\"",
           )
           expect(summary.attachments.map(&:filename)).to contain_exactly(
-            *[@secure_image, @secure_image_2, @secure_image_3].map(&:original_filename),
```

Comment:
`secure_image_2` is not in the excerpt of the first topic, so shouldn't be attached in the e-mail.